packages/vue/src/index.ts

```diff
@@ -0,0 +1,27 @@
+export function errorHandler(appsignal: any, Vue?: any) {
+  const DEFAULT_ACTION = "Vue.config.errorHandler"
+  const { version = "unknown" } = Vue
+
+  return function(error: Error, vm: any, info: string) {
+    const {
+      $vnode: {
+        componentOptions: { tag }
+      },
+      $options: { propsData }
+    } = vm
+
+    const span = appsignal.createSpan()
+
+    span
+      .setAction(tag || DEFAULT_ACTION)
+      .setTags({ framework: "Vue", info, version })
+      .setParams(propsData || {})
```

Comment:
Just realised something, in the API spec we treat params as a hash(map) of <string,string> does this work with the propsData here (e..g can we guarantee that a propsData object doesn't have values that are also objects?)

Reply:
Ah yeah, you're right, I've removed this option

Reply:
ok! Maybe we can create a helper at some point that ensures it's a key/value string/string hashmap :)